pkg/controller/performanceprofile/performanceprofile_controller_test.go

```diff
@@ -234,6 +234,50 @@ var _ = Describe("Controller", func() {
 			Expect(mcp.Spec.Paused).To(BeFalse())
 		})
 
+		It("should do nothing when pause annotation is set", func() {
```

Comment:
would be nice if this weren't between the "should do xyz on nth reconcile loop" tests

Reply:
yup, i can move it in a more logical place

Reply:
fixed, and I updated the logic to reflect your #40 changes